tests/dec/mages/mpk_archive_decoder_test.cc

```diff
@@ -0,0 +1,22 @@
+#include "dec/mages/mpk_archive_decoder.h"
+#include "test_support/catch.h"
+#include "test_support/decoder_support.h"
+#include "test_support/file_support.h"
+
+using namespace au;
+using namespace au::dec::mages;
+
+static const std::string test_archive = "tests/dec/mages/files/mpk/mini.mpk";
+
+TEST_CASE("MAGES. MPK archives archives", "[dec]")
+{
+    const std::vector<std::shared_ptr<io::File>> expected_files
+    {
+        tests::stub_file("WIN/test1.txt", "hello"_b),
+        tests::stub_file("test2.txt", "world"_b),
+    };
+    const auto decoder = MpkArchiveDecoder();
+    const auto input_file = tests::file_from_path(test_archive);
```

Comment:
Please consider preparing the file in the test itself to reduce number binary files.
(I know the repository contains many binary files, but I was too busy to fix this retroactively once it was decided to avoid them. The exception holds for files that are difficult to write encoder for, for example, TLG from Kirikiri - but that is not the case here).